### DISCUSSION TITLE
is nothing

### DISCUSSION THREAD

**Discussion by NidMo** [tailwindlabs/headlessui#1225] [Help]
when i install  @headlessui/vue

![image](https://user-images.githubusercontent.com/24955673/157582903-6bc5995a-2f89-4aad-bfe2-70a7f7d8cda4.png)

headlessui can't use in nuxt3 ?

**Comment by RobinMalfait** [comment]
The error seems to be unrelated to Headless UI. There are people using Headless UI in Nuxt.js so that shouldn't be a problem.

Can you create a reproduction repo and share it?
